src/pages/progress/index.tsx

```diff
@@ -57,110 +67,144 @@ export const OFFRAMPING_PHASE_SECONDS: Record<OfframpingPhase, number> = {
 
   return (
     <Box className="flex flex-col items-center justify-center mt-4">
       <div className="flex flex-col items-center justify-center max-w-[400px]">
         <WarningSection />
-        <div className="relative mt-12">
-          <svg
-            className="w-[200px] h-[200px] dark:text-white"
-            style={{ transform: 'scale()' }}
-            aria-hidden="true"
-            xmlns="http://www.w3.org/2000/svg"
-            fill="none"
-            viewBox="0 0 200 200"
-          >
-            <circle
-              cx="100"
-              cy="100"
-              r={CIRCLE_RADIUS}
-              stroke="rgb(132, 211, 245)"
-              strokeWidth={CIRCLE_STROKE_WIDTH}
-              style={{
-                strokeDasharray: `${((CIRCLE_RADIUS * 2 * Math.PI) / numberOfPhases) * targetPhaseIndex} ${
-                  CIRCLE_RADIUS * 2 * Math.PI
-                }`,
-                transition: `${OFFRAMPING_PHASE_SECONDS[currentPhase]}s linear`,
-                transform: 'rotate(-90deg)',
-                transformOrigin: '50% 50%',
-              }}
-            ></circle>
-            <circle
-              cx="100"
-              cy="100"
-              r={CIRCLE_RADIUS + CIRCLE_STROKE_WIDTH}
-              stroke="rgb(81, 184, 170)"
-              strokeWidth={CIRCLE_STROKE_WIDTH}
-            ></circle>
-          </svg>
-          <div className="absolute top-0 left-0 flex items-center justify-center w-full h-full text-4xl">
-            {currentPercentage}%
-          </div>
-        </div>
-        <h1 className="my-3 text-base font-bold text-blue-700">Your transaction is in progress.</h1>
-        <h1 className="mb-3 text-base text-blue-700">
+        <ProgressCircle
+          displayedPercentage={displayedPercentage}
+          showCheckmark={showCheckmark}
+          circumference={circumference}
+        />
+        <motion.h1
+          className="my-3 text-base font-bold text-blue-700"
+          initial={{ opacity: 0, y: 20 }}
+          animate={{ opacity: 1, y: 0 }}
+          transition={{ delay: 0.6 }}
+        >
+          Your transaction is in progress.
+        </motion.h1>
+        <motion.h1
+          className="mb-3 text-base text-blue-700"
+          initial={{ opacity: 0, y: 20 }}
+          animate={{ opacity: 1, y: 0 }}
+          transition={{ delay: 0.8 }}
+        >
           {!isNetworkEVM(selectedNetwork) ? 'This usually takes 4-6 minutes.' : 'This usually takes 6-8 minutes.'}
-        </h1>
-        <div>{message}</div>
+        </motion.h1>
+        <motion.div
+          className="flex flex-col items-center"
+          initial={{ opacity: 0, y: 20 }}
+          animate={{ opacity: 1, y: 0 }}
+          transition={{ delay: 0.9 }}
+        >
+          <div className="mb-2">{message}</div>
+          {remainingTime !== null && (
+            <p className="text-sm text-gray-500">Estimated time left: {remainingTime} seconds</p>
+          )}
+        </motion.div>
```

Comment:
Let's not show the remaining time per phase. We are only creating false expectations on the user. As long as the progress in the circle is moving, I don't really care how much each phase is going to take. 
Also, the phrasing is a little confusing as we say above that 'This usually takes 4-6 minutes' and here we say 'Estimated time left', which to me as a user means the estimated time of the whole process. 

To avoid any confusion, I'd opt to remove this detail altogether.